elevenclock/tools.py

```diff
@@ -410,11 +410,16 @@ def toggleClockVerPosAction():
         self.setContextMenu(menu)
         self.menuScreen = self.contextMenu().screen()
 
-        def reloadClocksIfRequired(reason: QSystemTrayIcon.ActivationReason) -> None:
-            if(reason != QSystemTrayIcon.ActivationReason.Context):
-                globals.restartClocks()
-
-        self.activated.connect(lambda r: reloadClocksIfRequired(r))
+        def activationHandler(reason: QSystemTrayIcon.ActivationReason) -> None:
+            match reason:
+                case QSystemTrayIcon.ActivationReason.DoubleClick:
+                    globals.sw.show()
+                case QSystemTrayIcon.ActivationReason.Context:
+                    return
+                case _:
+                    globals.restartClocks()
```

Comment:
@martinet101 is this really needed?

Reply:
Yes, (at least for me) when testing and changing settings files manyally it is very useful to be able to restart clocks with one click. And it is a fast way to reload the clocks if they bug out for whatever the reason (especially background colors and fs windows)